RadixWallet/Clients/SecureStorageClient/SecureStorageClient+Live.swift

```diff
@@ -489,6 +483,7 @@ let profileHeaderListKeychainKey: KeychainClient.Key = "profileHeaderList"
 private let deviceIdentifierKey: KeychainClient.Key = "deviceIdentifier"
 private let deviceInfoKey: KeychainClient.Key = "deviceInfo"
 private let p2pLinksKey: KeychainClient.Key = "p2pLinks"
+private let p2pLinksPKKey: KeychainClient.Key = "p2pLinksWalletPK"
```

Comment:
rename variable to `p2pLinksPrivateKey` and value to `PrivateKey`. `PK` can mean `Private` *or* `Public` key :)

Reply:
Right. I did name it this way the first time, but snyk scan failed with the error:
> Avoid hardcoding values that are meant to be secret. Found a hardcoded string used in here.

I guess it checks for the `PrivateKey` keyword.
Can we ignore this particular vulnerability?

Reply:
Oh I did not know.. silly snyk.

Call does SecretKey work? Otherwise check if as you might do an in-line snyk ignore comment?

Reply:
`SecretKey` does not work either.

What do you think about `p2pLinksWalletIdentity`? In fact, it's even more descriptive.